app/controllers/movies_controller.rb

```diff
@@ -12,13 +12,10 @@ def index
   end
 
   def show
-    show_response = movie_show_endpoint
-    cast_response = movie_cast_endpoint
-    review_response = movie_review_endpoint
-    @movie_object = Movie.new
-    @movie = parse(show_response)
-    @movie_cast = parse(cast_response)
-    @movie_review = parse(review_response)
+    @movie = MovieFacade.get_movie_details(params[:id])
+    # @movie = MovieFacade.movie_details(params[:id])
```

Comment:
There is some commented out code in this PR that I will clean up after we merge, and things look good!